package.json

```diff
@@ -30,13 +30,21 @@
   "contributes": {
     "languages": [
       {
-        "id": "sql",
+        "id": "pgsql",
```

Comment:
Should we keep it as just SQL? I think this is tricky, as most of the SQL files have ```.sql``` extension. Unless we can restrict it to ```.sql``` files next to the ```.pgproj```, I'd probably try to keep them generic.
I'm curious what's going to happen when we will have SSDT and our extension running side -by-side and they both going to contribute to ```.sql```? Maybe we should name our files ```.pgsql``` to be future proof?

Also, is there a command already to create new sql file in the project?

Reply:
This language id is already defined and if we use SQL, whenever we open a sql file it runs default syntax checking and we see a lot of syntax errors for those sql files as it is incorrect according to sql. We can actually name them pgsql, this would remove the overlap between them. 
No haven't added the create new sql file yet.

Reply:
What happens when they overlap? Which one wins? I guess we can do ```.pgsql```, but then there are other questions that arise from it:
1) What extension are we going use for the final script - ```.sql``` or ```.pgsql```? I think ```.sql``` is more commonly used, if they would want to ru it with third-party tools.
2) I think we will eventuzally collide in file extensions, no matter how hard we try and it will be a breaking change later. It may worth figuring out now how to treat ```.sql``` files differently depending on the folder where they are located.
We can definetely use ```pgsql``` id for the language, but the actual file extensions should be the same - ```.sql```.